vcd/resource_vcd_org_vdc_test.go

```diff
@@ -379,6 +527,8 @@ resource "vcd_org_vdc" "{{.VdcName}}" {
   enable_fast_provisioning = true
   delete_force             = true
   delete_recursive         = true
+  {{.FlexElasticKey}} {{.equalsChar}} {{.FlexElasticValue}}
```

Comment:
This line causes the HCL script to have bad formatting.
Please check the resulting script after `make test-binary-prepare`

Reply:
improved